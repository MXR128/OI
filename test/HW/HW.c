// Hello, world

#include <stdio.h>

int main() {
	// freopen("HW.out", "w", stdout);
	printf("Hello, world!\n");
	int a,b;
	scanf("%d%d",&a,&b);
	printf("%d",a+b);
	return 0;
}
