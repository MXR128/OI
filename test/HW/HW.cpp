#include <windows.h>

#include <iostream>

signed main() {
    // #ifndef ONLINE_JUDGE
    freopen("HW.in", "r", stdin);
    // #endif
    std::cout << "Hello, world!" << std::endl;

    int a, b;
    std::cin >> a >> b;
    std::cout << a + b;

    return 0;
}