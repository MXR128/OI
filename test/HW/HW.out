Hello, world!
Press any key to continue . . . 
2